src/middleware/externalRequests.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+const logger = require('../logger')
+
+exports.externalRequest = () => async (ctx, next) => {
```

Comment:
Lets rename this to `requestMiddleware` to keep it consistent with the other middleware naming. 
In theory, these request could also be internal. Sending a request to the mapper on a different endpoint to cary out a range of orchestrations

Reply:
Whoops. Rushed it :(